profile/profile.go

```diff
@@ -166,6 +168,7 @@ func (profile *Profile) parseConfig() error {
 	}
 
 	list, ok := profile.configPerspective.GetAsStringArray(CfgOptionEndpointsKey)
+	profile.endpoints = nil
 	if ok {
```

Comment:
`nil` is the default value. Is there a reason to set it explicitly?

Reply:
Yes, in case the profile config is changed and reloaded internally. There might be old data assigned.